Dockerfile

```diff
@@ -1,27 +1,8 @@
-FROM debian:jessie
+# Get the latest ccc alpine image. Started getting latest when version 1.0.6 was build.
+#FROM registry.ccctechcenter.org:5000/ccctechcenter/alpine-nginx-phpfpm-pgsql:1.0.6
+FROM ccctechcenter/alpine-nginx-phpfpm-pgsql:1.0.6
 
-MAINTAINER "Daniel McCoy" <danielmccoy@gmail.com>
-
-WORKDIR /tmp
-
-RUN apt-get update -y && \
-    apt-get install -y \
-    cron \
```

Comment:
The base image you're using donesn't have cron, so you'll still need to install it somewhere.   
```
10:47:06 ~$ docker run ccctechcenter/artisan-cron:1.0.6
Unable to find image 'ccctechcenter/artisan-cron:1.0.6' locally
1.0.6: Pulling from ccctechcenter/artisan-cron
cbdbe7a5bc2a: Already exists 
2fb9c0675eb7: Pull complete 
8f133914b4ae: Pull complete 
c3b7bb9c4720: Pull complete 
6297cb5bf1e7: Pull complete 
eb51395104ef: Pull complete 
673463a20c2c: Pull complete 
b998918098dd: Pull complete 
5757ff0793eb: Pull complete 
5eef81969190: Pull complete 
19d29d5fa94d: Pull complete 
ea0cbbb48277: Pull complete 
c936d168dfe6: Pull complete 
e5053965037f: Pull complete 
c233b3408057: Pull complete 
9806cfbecbcf: Pull complete 
1eca5eae0e3a: Pull complete 
9e5dce1d2398: Pull complete 
Digest: sha256:f2da28d3f247c17c46b339d98a1514b84f6d11845322ed8e87bcc18226cab62c
Status: Downloaded newer image for ccctechcenter/artisan-cron:1.0.6
/bin/sh: cron: not found
10:47:33 ~$ 

```

Reply:
Installed dcron via apk. Then tested that I coule run crontab -l in the resuntant instance.  I did not think to do that before.   Thanks for catching this.

I will have to wait until tomorrow afternoon to be sure cron actually does anything.